tests/utils_tests/test_restart_with_reloader.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+
+from django.test import SimpleTestCase, mock
+from django.utils import autoreload
+
+
+class TestEnvironmentGrooming(SimpleTestCase):
+    """
+    Checks for encoding conflicts in the environment variables when
+    autoreload is used.
+    """
+
+    def test_restart_with_reloader(self):
```

Comment:
```
def test_environment(self):
    """"
    With Python 2 on Windows, restart_with_reloader() coerces  environment
    variables to str to avoid "TypeError: environment can only contain
    strings" in Python's subprocess.py.
    """
```